### PR TITLE
Unify  DigitalTwin samples DTDL with C-SDK models

### PR DESCRIPTION
**Pull request by ridomin**
# Checklist
- [X] I have read the [contribution guidelines] (https://github.com/Azure/azure-iot-sdk-node/blob/master/.github/CONTRIBUTING.md).
- [ ] I added or modified the existing tests to cover the change (we do not allow our test coverage to go down).
- If this is a modification that impacts the behavior of a public API
  - [ ] I edited the corresponding document in the `devdoc` folder and added or modified requirements.

# Reference/Link to the issue solved with this PR (if any)

# Description of the problem

Samples (device and service) are not consistent with C-SDK samples.

# Description of the solution

Undo changes on DTDL files (we dont want to have different versions around)
Rename component names
Change commands to be lowercase

/cc @anthonyvercolano @YoDaMa @zolvarga 

/cc

### CODE REVIEW

**Comment by anthonyvercolano** [comment]
>   // https://docs.microsoft.com/en-us/azure/iot-hub/iot-hub-mqtt-support#respond-to-a-direct-method

This reference is useless.

---
Refers to: digitaltwins/samples/device/javascript/sample_device.js:59 in 0e4f37c. [](commit_id = 0e4f37c1c490cb1e60bfb62433564c67a9b54c87, deletion_comment = False)

File: digitaltwins/samples/device/typescript/src/sample_device.ts [hunk 1]
```diff
@@ -13,7 +13,6 @@
 import { DigitalTwinClient, CommandCallback, CommandRequest, CommandResponse, PropertyChangedCallback, BaseInterface } from 'azure-iot-digitaltwins-device';
 import { EnvironmentalSensor } from './environmentalinterface';
 import { DeviceInformation } from './deviceInformationInterface';
-import { SampleExit } from './exitInterface';
```
Comment (anthonyvercolano) [inline digitaltwins/samples/device/typescript/src/sample_device.ts:16]: This is unfortunate.  I rather liked this example.

**Comment by anthonyvercolano** [comment]
>     }, 5000);

The javascript example prints out a counter.  I thought that was a bit excessive.  This should at least say that it is sending telemetry every 5 seconds.

---
Refers to: digitaltwins/samples/device/typescript/src/sample_device.ts:93 in 0e4f37c. [](commit_id = 0e4f37c1c490cb1e60bfb62433564c67a9b54c87, deletion_comment = False)

**Comment by anthonyvercolano** [comment]
>   .catch(() => {console.log('the registration failed.');});

remove the .catch (and put a semicolon at the end of line 107.

---
Refers to: digitaltwins/samples/device/typescript/src/sample_device.ts:108 in 0e4f37c. [](commit_id = 0e4f37c1c490cb1e60bfb62433564c67a9b54c87, deletion_comment = False)

File: digitaltwins/samples/service/invoke_component_command.js [hunk 2]
```diff
@@ -6,9 +6,9 @@ const DigitalTwinServiceClient = require('azure-iot-digitaltwins-service').Digit
 
 const deviceId = process.env.IOTHUB_DEVICE_ID;
 const connectionString = process.env.IOTHUB_CONNECTION_STRING;
-const componentName = process.env.IOTHUB_COMPONENT_NAME; // for the environmental sensor, try "environmentalSensor"
-const commandName = process.env.IOTHUB_COMMAND_NAME; // for the environmental sensor, you can try "blink", "turnOff" or "turnOn"
-const commandPayload = process.env.IOTHUB_COMMAND_PAYLOAD; // for the environmental sensor, it really doesn't matter. any string will do.
+const componentName = process.env.IOTHUB_COMPONENT_NAME || 'sensor'; // for the environmental sensor, try "sensor"
+const commandName = process.env.IOTHUB_COMMAND_NAME || 'turnon'; // for the environmental sensor, you can try "blink", "turnoff" or "turnoff"
```
Comment (anthonyvercolano) [inline digitaltwins/samples/service/invoke_component_command.js:10]: >"turnoff [](start = 132, length = 8)

repeating.

File: digitaltwins/samples/device/typescript/src/sample_device.ts (hunk 1 above)
Comment (ridomin) [reply digitaltwins/samples/device/typescript/src/sample_device.ts:16]: All interfaces should be modeled with DTDL and them implemented for each language. We can discuss to add it in the future. What did you like about it?

Comment (anthonyvercolano) [reply digitaltwins/samples/device/typescript/src/sample_device.ts:16]: It was short, actionable, easy to understand, useful

---
In reply to: [413970433](https://github.com/Azure/azure-iot-sdk-node/pull/768#discussion_r413970433) [](ancestors = 413970433)

Comment (anthonyvercolano) [reply digitaltwins/samples/device/typescript/src/sample_device.ts:16]: But it looks like you don't want it so I'll close this.

---
In reply to: [414044301](https://github.com/Azure/azure-iot-sdk-node/pull/768#discussion_r414044301) [](ancestors = 414044301,413970433)

**Comment by anthonyvercolano** [comment]
>     }, 5000);

There should be a console.log message that says sending telemetry every 5 seconds.  OR you could do exactly what it is the javascript.  Please do one or the other.

---
In reply to: [618438632](https://github.com/Azure/azure-iot-sdk-node/pull/768#issuecomment-618438632) [](ancestors = 618438632)

---
Refers to: digitaltwins/samples/device/typescript/src/sample_device.ts:93 in 0e4f37c. [](commit_id = 0e4f37c1c490cb1e60bfb62433564c67a9b54c87, deletion_comment = False)

**Comment by anthonyvercolano** [comment]
Carried over with #773

**Pull request closed without merging**